discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patch-types.markdown

```diff
@@ -0,0 +1,46 @@
+# Patch types [](id=patch-types)
+
+Liferay ships Digital Enterprise fixes through three different channels. Fix packs and hotfixes. You can read about the differences in this document.
+
+## 1. Fix pack
+On a regular schedule, the latest fixes that patch the core are bundled together
+into fix packs, which are provided to all of Liferay's customers. Fix packs
+include fixes for both the core and the application suites that ship with the
+product. 
+
+## 2. Hotfix
+A hotfix is provided to a customer when they contact Liferay about an
+emergency situation, and Liferay's support team--working with the customer--
+determines that the problem is indeed an issue with the product that needs to be
+fixed very quickly. Support fixes the bug and provides a hotfix to the customer
+immediately. This is a short-term fix that solves the issue for the customer as
+quickly as possible. Hotfixes can patch both the core and the application
+suites.
+
+## Testing methods of patches
```

Comment:
We receive a lot of questions about testing this is why we added this information to the documentation. I've restructured it a bit and removed the details about unit and other types of testing.